src/pages/Admin/BackgroundSelect/BackgroundSelect.tsx

```diff
@@ -45,45 +45,66 @@ export const BackgroundSelect: React.FC<BackgroundSelectProps> = ({
 
   const hasBackgrounds = !!mapBackgrounds.length && !isLoadingBackgrounds;
 
+  const renderBackground = useCallback(
+    (mapBackground: string, index: number) => (
+      <button
+        className="BackgroundSelect__map"
+        disabled={isUploading}
+        style={{ backgroundImage: `url(${mapBackground})` }}
+        key={index}
+        onClick={() => uploadMapBackground(mapBackground)}
+      />
+    ),
+    [isUploading, uploadMapBackground]
+  );
+
   const renderMapBackgrounds = useMemo(
     () =>
-      mapBackgrounds.map((mapBackground, index) => (
-        <button
-          className="BackgroundSelect__map"
-          disabled={isUploading}
-          style={{ backgroundImage: `url(${mapBackground})` }}
-          key={index}
-          onClick={() => uploadMapBackground(mapBackground)}
-        />
-      )),
-    [isUploading, mapBackgrounds, uploadMapBackground]
+      mapBackgrounds.map((mapBackground, index) =>
+        renderBackground(mapBackground, index)
+      ),
+    [mapBackgrounds, renderBackground]
+  );
+
+  const renderDefaultBackgrounds = useMemo(
+    () =>
+      DEFAULT_BACKGROUNDS.map((mapBackground, index) =>
+        renderBackground(mapBackground, index)
+      ),
+    [renderBackground]
   );
 
   return (
     <div className="BackgroundSelect">
-      {!mapBackground && (
-        <>
-          <FileButton
-            disabled={isUploading}
-            title="Import a map background"
-            description="Recommended size: 2000px / 1200px"
-            onChange={uploadMapBackground}
-          />
-
-          <h3 className="BackgroundSelect__maps-header">
-            Or select one of our map backgrounds
-          </h3>
-          {isLoadingBackgrounds && <div>Loading maps...</div>}
-
-          <div className="BackgroundSelect__map-grid">
-            {hasBackgrounds && renderMapBackgrounds}
-          </div>
-        </>
-      )}
-
-      {mapBackground && (
-        <img width="100%" src={mapBackground} alt="Venue map background" />
-      )}
+      <>
+        <FileButton
+          disabled={isUploading}
+          title="Import a map background"
+          description="Recommended size: 2000px / 1200px"
+          onChange={uploadMapBackground}
+        />
+
+        <h3 className="BackgroundSelect__maps-header">
+          Or select one of our map backgrounds
+        </h3>
+
+        <div className="BackgroundSelect__map-grid">
+          {renderDefaultBackgrounds}
+          {hasBackgrounds && renderMapBackgrounds}
+        </div>
+
+        {isLoadingBackgrounds && <div>Loading maps...</div>}
```

Comment:
`<Loading label="loading maps..." />` ?

Reply:
This component renders an image, we need just a plain text here.